packages/command-bundle/Listener/CommandFlowListener.php

```diff
@@ -42,9 +51,41 @@ public static function getSubscribedEvents(): array
         ];
     }
 
-    public function __construct(Connection $executionConnection)
-    {
+    public function __construct(
+        Connection $executionConnection,
+        EventDispatcherInterface $eventDispatcher
+    ) {
         $this->connection = $executionConnection;
+        $this->eventDispatcher = $eventDispatcher;
+    }
+
+    /**
+     * This is a fallback on the compiler pass system to be sure options are available if command are registered
+     * by another mean.
+     */
+    public function addOptions(ConsoleCommandEvent $consoleCommandEvent): void
```

Comment:
@m097 Does it work if we only rely on the compiler pass ? I don't want to have code that is not needed... It's why I did remove it on the first place.